### PR TITLE
feat(keyboard): support simple cut-pasting using meta+x/v

### PR DESCRIPTION
**Pull request by luin**
Similar to how we handle Meta+c, in this PR we support Meta+x so users can cut text on the page.

References #18755

### CODE REVIEW

**Comment by luin** [comment]
@microsoft-github-policy-service agree

File: tests/page/page-keyboard.spec.ts [hunk 1]
```diff
@@ -482,6 +482,17 @@ it('should support simple copy-pasting', async ({ page, isMac, browserName }) =>
   expect(await page.evaluate(() => document.querySelector('div').textContent)).toBe('123123');
 });
 
+it('should support simple cut-pasting', async ({ page, isMac, browserName }) => {
```
Comment (ErnstdeJong) [inline tests/page/page-keyboard.spec.ts:485]: why is browserName called here? it's not used

Comment (luin) [reply tests/page/page-keyboard.spec.ts]: Good catch. Updated.

Comment (luin) [reply tests/page/page-keyboard.spec.ts]: cc @dgozman not sure why re-requesting review removed you from the list. Don't think I have permission to add you so ping manually.

**Review by pavelfeldman — APPROVED** [review]

**Pull request merged**